Jenkinsfile

```diff
@@ -29,8 +29,13 @@ def generatePackageStep(opts, arch) {
         node("linux&&${arch}") {
             stage("${opts.image}-${arch}") {
                 try {
-                    checkout scm
+                    sh 'docker version'
+                    sh 'docker info'
+                    sh '''
```

Comment:
nit: this item is displayed as "Shell script" in the Jenkins logs, which is not very descriptive... Is there a way to add something like a title ?

Reply:
I think there's a third-party plugin to add labels, but don't think it's in Jenkins code (looked at that in moby/moby, but we decided not to use it and stick to core functionality where possible)